tests/docs/introduction.spec.ts

```diff
@@ -1,29 +1,18 @@
 /* eslint-disable @typescript-eslint/no-empty-function */
 import { expect, test } from '@playwright/test';
 
-test('docs header element is active', async ({ page }) => {
+test('Docs header element is active', async ({ page }) => {
   // navigate to the docs landing page /docs
   await page.goto('/docs');
+
   // check if the docs link in the page header is having the active class
   await expect(page.getByRole('link', { name: 'Docs' })).toHaveClass(/active/);
 });
 
-test('Correct Page Heading', async ({ page }) => {
+test('Page Heading is correct', async ({ page }) => {
   // navigate to the docs landing page /docs
   await page.goto('/docs');
+
   // check if the heading contains Introduction
   expect(await page.textContent('h1')).toBe('Introduction');
 });
-
-test.fixme(
-  'is the left-side menu containing an element with the same name as the page headline',
-  async ({ page }) => {
-    // navigate to the docs landing page /docs
-    // check if the Introduction element in the left menu is holding the active class
-  },
-);
-
-test.fixme('is the highlighted item in the left-menu is the correct one', async ({ page }) => {
-  // navigate to the docs landing page /docs
-  // check if the Introduction element in the left menu is holding the active class
```

Comment:
Ohh I think the test case is valid, you mean the test description is the same one for the ones above right? In case yes, we should definitely check the steps for testing it!

Reply:
@Cahllagerfeld : well, the 27-28 are duplicates of lines 21-22, they say the very same thing:
```
  // navigate to the docs landing page /docs
  // check if the Introduction element in the left menu is holding the active class
```

Reply:
Could you confirm the proper steps in those skeletons should be as follows?

```
test.fixme('is the highlighted item in the left-menu is the correct one', async ({ page }) => {
  // navigate to the docs landing page /docs
  // check if the Introduction element in the left menu is highlighted element
});
```

```
test.fixme(
  'is the left-side menu containing an element with the same name as the page headline',
  async ({ page }) => {
    // navigate to the docs landing page /docs
    // check if both element in the left menu and page header are the same and say 'Introduction'
  },
);
```